### ISSUE TITLE
Same interpolation object with multiple getFixedT() in different locales yields wrong translation

### ISSUE THREAD

**Issue by nielskrijger** [i18next/i18next#851]
Hi guys,

Thanks for the great library. I was using the old i18next and recently upgraded to the new version. Looks great! Much cleaner and better documented, love it.

I came across some unexpected behaviour; the following test setup demonstrates the issue:

```js
const i18next = require('i18next');

i18next.init({
  lng: 'en',
  resources: {
    en: {
      translation: {
        key1: 'EN {{var}}'
      }
    },
    nl: {
      translation: {
        key1: 'NL {{var}}'
      }
    }
  }
}, (err, t) => {
  const english = i18next.getFixedT('en');
  const dutch = i18next.getFixedT('nl');

  // This prints "EN foo NL foo" as expected
  console.log(english('key1', { var: 'foo' }), dutch('key1', { var: 'foo' }));

  // This prints "EN foo EN foo", which is unexpected
  const interpolation = {
    var: 'foo'
  }
  console.log(english('key1', interpolation), dutch('key1', interpolation));
});
```

The behaviour surprised me and took some time to debug. I'm guessing some caching is going on somewhere. I'm not familair with the internals of i18next.

I uploaded the test setup here: https://github.com/nielskrijger/i18next-test

**Comment by jamuhl** [comment]
https://github.com/i18next/i18next/blob/master/src/i18next.js#L212

rather clear what's going wrong...bad me do mess with the options param...setting lng on it.

will be fixed asap and a patch version will be released...sorry for this.

**Comment by jamuhl** [comment]
should be fixed in i18next@4.1.2

**Comment by nielskrijger** [comment]
Wow, that was fast.

Checked and works! Thank you.
